include/blogPost.php

```diff
@@ -0,0 +1,48 @@
+<?php
+function getArray(){
+    $post = array(
+        array(
+            'postid'=>0,
+            'title'=> "Getting To Know Me <br/>",
+            'body'=> "<div class = 'aboutMe'>
+                    Below are some more facts about me: <br/> <br/>
+                    I am from NY <br/>
+                     My favorite dessert is ice cream <br/>
+                     My least favorite forrd is chocolate <br/>
+                     I am scared of cats <br/>
+             My favorite Disney movie is mulan <br/><br/><br/>"
+         ),
+         array(
+             'postid'=>1,
+             'title'=> "What I am Studying <br/>",
+             'body'=> "<div class = 'aboutMe'>
+                     I am currently studying CogNeuro<br/><br/><br/>",
+          ));
+          return $post;
+}
+
+function getPost(){
+    $post = getArray();
+    for ($i = 0; $i<count($post); $i++){
+        $postId = $post[$i]['postid'];
+        $postTitle = $post[$i]['title'];
+        $postBody= $post[$i]['body'];
+      echo  "<a class = 'resume' href = 'viewPost.php?postid=". $postId."'>".$postTitle." </a><br/>";
+    }
+}
+
+
+
+function blog(){
+    $post = getArray();
+    $postId = $_REQUEST['postid'];
```

Comment:
It's generally best not to access `$_REQUEST` from within a function. Instead, have this function take a `$postId` parameter, and pass that in when you call the post in `viewPost.php`. That way, this function can be used in other contexts.